src/debug/HyprCtl.cpp

```diff
@@ -59,12 +60,13 @@ std::string monitorsRequest(HyprCtl::eHyprCtlOutputFormat format) {
         for (auto& m : g_pCompositor->m_vMonitors) {
             result += getFormat("Monitor %s (ID %i):\n\t%ix%i@%f at %ix%i\n\tdescription: %s\n\tmake: %s\n\tmodel: %s\n\tserial: %s\n\tactive workspace: %i (%s)\n\treserved: %i "
                                 "%i %i %i\n\tscale: %.2f\n\ttransform: "
-                                "%i\n\tfocused: %s\n\tdpmsStatus: %i\n\n",
+                                "%i\n\tfocused: %s\n\tdpmsStatus: %i\n\tvariable refresh rate: %s\n\n",
```

Comment:
two notes: no spaces in name (vrr better) and it should be `0/1` and not `yes/no`

Reply:
Done in 9bcfb5079a404276d43df6792a559efb2938b4bc